synchronization/src/androidTest/java/de/cyface/synchronization/TestUtils.java

```diff
@@ -25,14 +25,14 @@
     /**
      * An username used by the tests to set up a Cyface account for synchronization.
      */
-    final static String DEFAULT_USERNAME = "test";
+    final static String DEFAULT_USERNAME = "test@cyface.de";
     /**
      * A password used by the tests to set up a Cyface account for synchronization.
      */
-    final static String DEFAULT_PASSWORD = "secret";
+    final static String DEFAULT_PASSWORD = "7vXnwVAXfWXKbC32";
     /**
      * Path to an API available for testing.
      */
     @SuppressWarnings("unused") // because this is used in the cyface flavour
-    final static String TEST_API_URL = "https://s1.cyface.de/api/v2";
+    final static String TEST_API_URL = "https://s2.cyface.de/api/v2";
```

Comment:
s2 should not be used for testing

Reply:
The problem is that we have some (old) tests which still rely on an actual API although (from the code) they should use mocked code. In order to be able to fix the bug in the current releases I need to keep the CI happy.

Thus, I am using the s2 but with a special test account which we can clean up easily.

**Another option** would be to disable all API dependent tests until we have time to investigate this dependency (which we did already some time ago but without sustainable results).

Reply:
Disable them and create a task.